packages/terra-clinical-label-value-view/src/LabelValueView.scss

```diff
@@ -7,7 +7,7 @@
   }
 
   .label {
-    color: #75767d;
+    color: var(--terra-label-value-label-color, #75767d);
     font-size: 0.8571428571rem;
```

Comment:
Should we open up font-size, line-height, and margin-bottom to be themeable too?

Reply:
Scope of this PR is limited to label text color.

Reply:
There's potential for other consumers to want to customize these values. Since this is the first run through to create themeable variables on clinical, it'd be helpful to discuss which values actually need to be themeable. Tagging @neilpfeiffer for input. If we do, we can log another issue so it's not a blocker.